views/lexicon/forms/toggle-switch.html.twig

```diff
@@ -61,8 +61,8 @@
         form.toggle_switch({
             'id': 'toggle-6',
             'label': 'Toggle switch with a longer than expected label which will probably wrap multple lines and push the following input onto the next baseline',
-            'help': 'Example block-level help text here',
-            'guidance': 'Help text to give more information about expected input'
+            'guidance': 'Help text to give more information about expected input',
+            'help': 'Example block-level help text here'
```

Comment:
Curious as to why you needed to change the order here?

Reply:
It was a restoration of the code. No real difference

Reply:
As for the styles, I have a separate ticket (#641) to work those details separately